frontend/src/metabase/dashboard/components/DashCard/DashCardActionButtons/DashCardActionButton/DashCardActionButton.tsx

```diff
@@ -9,22 +9,22 @@ import {
 } from "./DashCardActionButton.styled";
 
 interface Props extends React.HTMLAttributes<HTMLAnchorElement> {
-  tooltip: string;
+  tooltip?: string;
   analyticsEvent?: string;
 }
 
-function DashActionButton({
-  tooltip,
-  analyticsEvent,
-  children,
-  ...props
-}: Props) {
-  return (
-    <StyledAnchor {...props} data-metabase-event={analyticsEvent}>
-      <Tooltip tooltip={tooltip}>{children}</Tooltip>
-    </StyledAnchor>
-  );
-}
+const DashActionButton = React.forwardRef<HTMLAnchorElement, Props>(
```

Comment:
Mantine requires components used as Menu.Trigger to accept a ref: https://mantine.dev/core/menu/#required-ref-prop